samples/Directory.Build.targets

```diff
@@ -7,6 +7,7 @@
   <Import Project="$(MSBuildThisFileDirectory)..\src\Sentry.Bindings.Cocoa\buildTransitive\Sentry.Bindings.Cocoa.targets"
     Condition="'$(OutputType)' == 'Exe' And ('$(TargetPlatformIdentifier)' == 'ios' Or '$(TargetPlatformIdentifier)' == 'maccatalyst')" />
   <Import Project="$(MSBuildThisFileDirectory)..\src\Sentry\Platforms\Native\buildTransitive\Sentry.Native.targets" />
+  <Import Project="$(MSBuildThisFileDirectory)..\src\Sentry\buildTransitive\Sentry.SourceGenerators.targets" />
```

Comment:
note: We deleted `Sentry.SourceGenerators.targets` in #4333.

It was redundant in the NuGet package, as only `Sentry.props` and `Sentry.targets` are automatically picked up in consuming projects (via `PackageReference`), and `Sentry.SourceGenerators.targets` was never `Import`ed in any of these MSBuild files either.